docs/examples/custom_operations/numba_function.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<p>presents  -&gt; shows  (<em>we</em> present that the tutorial)</p><p> </p><p>user function -&gt; user-defined function</p><p> </p><p>foreign C code -&gt; external C code</p><p> </p><p>utils -&gt; utilities  (no jargon)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>custom operations written as <s>a </s>Numba JIT-compiled function<strong>s</strong>.</p><p> </p>